src/find.js

```diff
@@ -51,7 +52,8 @@ module.exports = async function(collection, params) {
    *
    * See mongo documentation: https://docs.mongodb.com/manual/reference/collation/#collation-and-index-use
    */
-  const collatedQuery = config.COLLATION ? query.collation(config.COLLATION) : query;
+  const collation = params.collation || config.COLLATION;
```

Comment:
Nice enhancement!

What do you think about also providing the ability to turn _off_ collation for a specific query?

```
// Global collation set up
paging.config.COLLATION = { locale: 'en' };

// Turn collation off for a single query
const results = await paging.find(collection, { ..., collation: undefined })
```

Reply:
Done at https://github.com/mixmaxhq/mongo-cursor-pagination/pull/320/commits/c2ff6dae5824820414d5286f91d0cd7eedf0ba90